src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -152,11 +164,17 @@ def get(self, request: Request, project: Project) -> Response:
         )
         matched_code_mappings = []
         with configure_scope() as scope:
-            set_top_tags(scope, project)
+            set_top_tags(scope, project, ctx, len(configs) > 0)
             for config in configs:
                 if not filepath.startswith(config.stack_root) and not mobile_frame:
                     result["error"] = "stack_root_mismatch"
                     continue
+                # XXX: The logic above allows all code mappings to be processed
+                if (
+                    filepath.startswith(config.stack_root)
+                    and config.automatically_generated is True
+                ):
+                    scope.set_tag("stacktrace_link.automatically_generated", True)
```

Comment:
This will help determine the performance of automatically generated code mappings.